pkg/testers/ginkgo/ginkgo.go

```diff
@@ -94,9 +96,43 @@ func (t *Tester) pretestSetup() error {
 	}
 	log.Printf("Using kubeconfig at %s", t.kubeconfigPath)
 
+	if err := t.AcquireTestPackage(); err != nil {
+		return fmt.Errorf("failed to get ginkgo test package from published releases: %s", err)
+	}
+
 	return nil
 }
 
+// TODO(michaelmdresser): change behavior if a local built e2e.test package
+// is available
+func (t *Tester) AcquireTestPackage() error {
+	releaseTar := fmt.Sprintf("kubernetes-test-%s-%s.tar.gz", runtime.GOOS, runtime.GOARCH)
+
+	script := []string{
+		"set -o xtrace",
+		"set -o pipefail",
+		"set -o errexit",
+		"set -o nounset",
+		"cd $ARTIFACTS",
+		fmt.Sprintf(
+			"gsutil cp gs://kubernetes-release/release/$(gsutil cat gs://kubernetes-release/release/latest.txt)/%s .",
+			releaseTar),
+		fmt.Sprintf(
+			"tar -xzf %s",
```

Comment:
can we untar by using standard golang packages ? 
https://golang.org/pkg/archive/tar/
https://golang.org/pkg/compress/gzip/

Reply:
Done. Specifically, it will now only extract the file we care about and nothing else.